Jellyfin.Server.Implementations/FullSystemBackup/BackupService.cs

```diff
@@ -0,0 +1,396 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using System.Text.Json.Serialization;
+using System.Threading;
+using System.Threading.Tasks;
+using Jellyfin.Database.Implementations;
+using MediaBrowser.Common.Configuration;
+using MediaBrowser.Controller;
+using MediaBrowser.Controller.SystemBackupService;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+
+namespace Jellyfin.Server.Implementations.Backup;
```

Comment:
```suggestion
namespace Jellyfin.Server.Implementations.FullSystemBackup;
```

Will need to update namespaces for everything in this folder (or rename the folder)

Reply:
A bit of context here, `Backup` as a folder name is in the gitignore so i would rather have named it that instead although i dont want to remove that from the gitignore in case i break something else.
Still change?